Makefile

```diff
@@ -246,6 +246,16 @@ ENVTEST = $(shell pwd)/bin/setup-envtest
 envtest: ## Download envtest-setup locally if necessary.
 	$(call go-install-tool,$(ENVTEST),sigs.k8s.io/controller-runtime/tools/setup-envtest@latest)
 
+CMCTL = $(shell pwd)/bin/cmctl
+OS = $(shell go env GOOS)
+ARCH = $(shell go env GOARCH)
+.PHONY: cmctl
+cmctl: ## Download cmctl locally if necessary.
+ifeq ("$(wildcard $(CMCTL))","")
```

Comment:
I'm preferring to use go install: 
`go install github.com/oleewere/cmctl@v0.5.0`

Reply:
I'm following the installation steps as documented in the tools documentation.
Furthermore, while it's name is the same, the repo you provided is a totally different tool that has nothing to do with the certificate manager

Reply:
Yes, I saw those instructions but I tried to find a go install cmd for cmctl.
So I guess It's fine by me